source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -377,11 +383,20 @@
                                 }
                             }
                         }
+                        valuesByItem.hasComments = ko.computed(function () {
+                            return _.some(valuesByItem.comments, function (comment) {
+                                return comment.HasComment() && comment.IsCalificationShown();
+                            });
+                        });
                         valuesByItem.hasSelfComments = ko.computed(function () {
                             return _.some(valuesByItem.comments, function (comment) {
                                 return comment.HasComment() && comment.isSelfComment;
                             });
                         });
+                        valuesByItem.areCommentsEditable = ko.computed(function () {
+                            return viewmodel.isEvaluationEditable();
```

Comment:
Since the result does not depend in the item, is `areCommentsEditable` really necessary? what do you think about to directly use `viewmodel.isEvaluationEditable` in the view?